RecoLocalCalo/HcalRecAlgos/src/MahiFit.cc

```diff
@@ -69,16 +69,17 @@ void MahiFit::phase1Apply(const HBHEChannelInfo& channelData,
     auto const pedWidth = channelData.tsPedestalWidth(iTS);
 
     //Photostatistics
-    auto const noisePhoto = (amplitude > pedWidth) ? std::sqrt(amplitude * channelData.fcByPE()) : 0.f;
+    auto const noisePhotoSq = (amplitude > pedWidth) ? (amplitude * channelData.fcByPE()) : 0.f;
```

Comment:
do I understand correctly that this is the only place where numerical changes are expected for Run2/Run1 workflows?

Here is an example change in chi2 of the endcap hits with e>1 MeV in wf 136.874 (2018 EGamma).
![all_OldVSNew_RunEGamma2018Cwf136p874c_log10HBHERecHitsSorted_hbhereco__reRECO_obj_obj_chi223](https://user-images.githubusercontent.com/4676718/94215946-f22df580-fe92-11ea-84ef-ed798c15bb0d.png)

or are the other changes like .noalias or .lazyProduct expected to change the order of operands which may lead to some bitwise differences?

Reply:
Yes, I think this is the only place where small numerical changes occur. Changes like .noalias and .lazyProduct would only affect on the speed of matrix operations.

Reply:
> Here is an example change in chi2 of the endcap hits with e>1 MeV in wf 136.874 (2018 EGamma).

I looked manually hit by hit and about 1/3 of the hits have a numerical difference in the energy. However, out of 161K hits with differences, only 12 are above 1E-5 and none are above 1E-4.  
This is just to confirm that the downstream differences in objects like PF candidates and jets are just a butterfly effect of small numerical differences.

Reply:
It is good not to have a big numerical difference. This numerical difference is not intended, but because the numerical difference comes from redundant calculation, taking the square of the sqrt of the value, I think the current result is more reasonable.